rerun_js/scripts/version.mjs

```diff
@@ -56,8 +58,18 @@ for (const pkg_path of root_package_json.workspaces) {
       /<https:\/\/app\.rerun\.io\/.*\/examples\/dna\.rrd>/,
       `<https://app.rerun.io/version/${version}/examples/dna.rrd>`,
     );
+    if (index_ts) {
```

Comment:
don't understand what this is all about. We want to load the ts file because we need it for generating docs but it has a dna.rrd path in there that we have to reformat?

Reply:
I want the versioned link to the DNA example in the `index.ts` file to be updated automatically when our release workflow versions JS stuff